### ISSUE TITLE
What should be done on captcha expire?

### ISSUE THREAD

**Issue by gaurishhs** [marsidev/react-turnstile#3]
If the captcha expires, How can i bring up a new captcha instead of telling the user to reload the page

**Comment by marsidev** [comment]
Hi. A token is valid for 300 seconds ([ref](https://developers.cloudflare.com/turnstile/frequently-asked-questions/#how-long-is-a-turnstile-token-valid-for-before-it-is-rejected-by-siteverify)). 

The intended behavior of this lib is auto-reset the widget every 250 seconds, so you never would have an expired token. You can see it [here](https://github.com/marsidev/react-turnstile/blob/bc94236a25d70e537a4a94ea63e5fcb01ad2813d/src/lib.tsx#L123-L127).

But there is a typo which causes the auto-reset to happening every 62.5 seconds. But still you never should have an expired token. Are you experiencing this?

Anyways, I'm planning to add a property called `autoResetOnExpire` to indicate whether the widget should automatically reset when it expires (every 300 seconds) or manually handling the expiring. This is mentioned in the [docs](https://developers.cloudflare.com/turnstile/frequently-asked-questions/#what-happens-if-the-user-takes-longer-than-five-minutes).

For a manual handling you need to call the `.reset()` method in the `onExpire` callback. Something like this:

```jsx
import { useRef } from 'react'
import { Turnstile } from '@marsidev/react-turnstile'

function Widget() {
  const ref = useRef(null)

  function onExpire() {
    ref.current?.reset()
  }

  return (
    <Turnstile ref={ref} siteKey={process.env.SITEKEY} onExpire={onExpire}  />
  )
}
```

> Note that invoking the `.reset()` method only resets the turnstile widget and not the entire page.

**Comment by marsidev** [comment]
Resolved at [447f979](https://github.com/marsidev/react-turnstile/commit/447f979904ba31279728e743680d5a561a926c87). Check the [readme](https://github.com/marsidev/react-turnstile#handling-widget-expiring) for a example handling the expiring manually.
